cubedash/_utils.py

```diff
@@ -415,6 +437,19 @@ def dataset_shape(ds: Dataset) -> Tuple[Optional[Polygon], bool]:
     return geom, True
 
 
+def to_wrs_84(geometry: BaseGeometry, crs: Union[int, str, dict, pyproj.CRS]):
+    origin = pyproj.Proj(init=crs)
+    dest = pyproj.Proj(init="epsg:4326")
+    tranform_wrs84 = functools.partial(pyproj.transform, origin, dest)
```

Comment:
There are datacube functions that do this, which are faster, somehow!

See: https://github.com/opendatacube/odc-tools/blob/develop/libs/index/odc/index/stac.py#L76

Reply:
Thanks -- that api is much cleaner than doing it ourselves.

Added.